.coderabbit.yaml

```
@@ -0,0 +1,40 @@
+      - "*.yml"
+      - "*.toml"
+      - "*.json"
+    settings:
+      check_syntax: true
+chat:
+  auto_reply: true
+  create_issues: true
+knowledge_base: {}
+code_generation:
+  docstrings:
+    language: en-US
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)